main.tf

```diff
@@ -107,6 +107,24 @@ module "database" {
   ]
 }
 
+module "certificates" {
+  source = "./modules/certificates"
+
+  install_cert_manager           = var.install_cert_manager
+  use_self_signed_cluster_issuer = var.use_self_signed_cluster_issuer
+  cert_manager_namespace         = var.cert_manager_namespace
+  name_prefix                    = local.name_prefix
+
+  depends_on = [
+    module.eks,
+    # The AWS LBC installs webhooks, and all other K8S stuff can fail
+    # if they are deployed, but the AWS LBC pods aren't up yet.
+    # This doesn't actually need the LBC,
+    # but we want to avoid concurrently updating these.
+    module.aws_lbc,
+  ]
+}
+
 module "operator" {
   source = "github.com/MaterializeInc/terraform-helm-materialize?ref=v0.1.8"
```

Comment:
Will this line change if we're waiting on a new helm release?

Reply:
Yes, I think so.